internal/proto/controller/api/resources/targets/v1/target.proto

```diff
@@ -170,18 +170,13 @@ message Target {
     }
   ]; // @gotags: `class:"public"`
 
-  // Output only. The IDs of the application credential source ids associated with this Target.
   // Deprecated use "brokered_credential_source_ids" instead.
-  repeated string application_credential_source_ids = 400 [
-    json_name = "application_credential_source_ids",
-    deprecated = true
-  ]; // @gotags: `class:"public"`
-  // Output only. The application credential sources associated with this Target.
+  reserved "application_credential_source_ids";
```

Comment:
I believe the standard is to include all these reserved fields at the bottom of this Message under the `// Deprecated fields` comment